TextFieldEffects/TextFieldEffects/TextFieldEffects.swift

```diff
@@ -10,20 +10,20 @@ import UIKit
 
 extension String {
     /**
-    true iff self contains characters.
+    true if self contains characters.
```

Comment:
Although this is valid in programming "if and only if".  It has confused people already :)